tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -77,8 +55,8 @@ setup() {
     echo "Checking the image was pulled in the guest"
     sandbox_id=$(get_node_kata_sandbox_id $node)
     echo "sandbox_id is: $sandbox_id"
-    # With annotation for nydus, only rootfs for pause container can be found on host
-    assert_rootfs_count "$node" "$sandbox_id" "1"
+    # No rootfs can be found on host with guest pull
+    assert_rootfs_count "$node" "$sandbox_id" "0"
```

Comment:
We could consider removing the `assert_rootfs_count` checks entirely, as we are running with sharedfs=none now?

Reply:
Done.